README.testsuite.md

```diff
@@ -1,74 +1,146 @@
+ - [Tests](https://github.com/sqlalchemy/sqlalchemy/tree/main/lib/sqlalchemy/testing/suite)
 
- - **Unit Tests**
-     - [Readme](https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst)
-     - [Tests](https://github.com/sqlalchemy/sqlalchemy/tree/main/test)
+**Unit Tests**
+ - Separate from the dialect compliance suite.
+ - A large portion of these tests do not have dialect-sensitive functionality.
+ - SQLAlchemy version 2.0.36 contains over 30K unique tests.
+ - [Readme](https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst)
+ - [Tests](https://github.com/sqlalchemy/sqlalchemy/tree/main/test)
 
-The dialect compliance suite was initially added to SQLAlchemy version 0.8 in 2012 via commit [568de1e](https://github.com/sqlalchemy/sqlalchemy/blob/568de1ef4941dcf366d81ebb46e122f4a973d15a/README.dialects.rst).
+## Test Execution
 
-In order to run the dialect compliance suite, it is necessary that dialects have a `requirements.py`. A `requirements.py` file was added to SQLAlchemy Ingres connector 0.0.7 in May 2024 via PR [49](https://github.com/ActianCorp/sqlalchemy-ingres/pull/49).
-
-The SQLAlchemy unit tests are separate from the dialect compliance suite. The number of tests varies depending on the exact version of SQLAlchemy. For recent releases of SQLAlchemy 2.x, there are over 30K unit tests while the dialect compliance suite contains fewer than 2K tests.
-
-In PR [42](https://github.com/ActianCorp/sqlalchemy-ingres/pull/42), changes were made to the SQLAlchemy Ingres connector that included new import statements. These changes broke compatibility with SQLAlchemy 1.4. PR [69](https://github.com/ActianCorp/sqlalchemy-ingres/pull/69) fixed this problem so that the Ingres connector once again works with SQLAlchemy 1.x, retains compatibility with SQLAlchemy 2.x, and is able to run the SQLAlchemy dialect compliance suite and the unit tests for SQLAlchemy versions 1.x and 2.x.
-
-
-# SQLAlchemy Dialect Compliance Suite
-## Overview and Setup
-
-The following text is from the page [README.dialects.rst](https://github.com/sqlalchemy/sqlalchemy/blob/main/README.dialects.rst) of the SQLAlchemy GitHub repository.
-
-> SQLAlchemy includes a [dialect compliance suite](https://github.com/sqlalchemy/sqlalchemy/tree/main/lib/sqlalchemy/testing/suite) that is usable by third party libraries, in the source tree at lib/sqlalchemy/testing/suite. There's no need for a third party dialect to run through SQLAlchemy's full testing suite, as a large portion of these tests do not have dialect-sensitive functionality. The "dialect compliance suite" should be viewed as the primary target for new dialects.
-
-### Environment and Packages
-These steps have been tested using the following environment:
+### Example Environment and Packages
+The operations described in this document have been tested in various environments up to and including the following versions:
```

Comment:
this is going to introduce a doc update burden - I'm not sure if this is worth the time spent on paperwork. I'm not against this per-se but I'm not for it.

Reply:
👍 Works for me 😸